src/web/src/student-quotes.js

```diff
@@ -25,8 +25,7 @@ export default [
   },
   {
     author: 'Abdulbasid Guled',
-    url:
-      'https://dev.to/hyperthd/4-months-of-open-source-what-i-ve-learned-from-taking-dps909-and-what-i-plan-to-do-about-dps911-next-month-1g62',
+    url: 'https://dev.to/hyperthd/4-months-of-open-source-what-i-ve-learned-from-taking-dps909-and-what-i-plan-to-do-about-dps911-next-month-1g62',
```

Comment:
Unnecessary changes, revert this whole file.

Reply:
when pushing the new commit to GitHub, the system autotest. If I don't change it, it can't be uploaded.

Reply:
What do you mean?

Reply:
I worked on the dashboard recently and I didn't have to make changes to these files.

Reply:
no worry, if the issue reproduced next time, I will post screen shot